site/pages/gc-form-en.hbs

```diff
@@ -0,0 +1,197 @@
+		</div>
+		<div class="col-md-4">
+			<h5 class="mrgn-tp-0">Code</h5>
+			<pre class="prettyprint prettyprinted" style=""><code><span class="com">// Large checkboxes:</span>&#x3C;ul class=&#x22;list-unstyled gc-form&#x22;&#x3E;
+	&#x3C;li class=&#x22;checkbox&#x22;&#x3E;
+		&#x3C;input type=&#x22;checkbox&#x22; id=&#x22;checkbox_1&#x22;&#x3E;
+		&#x3C;label for=&#x22;checkbox_1&#x22;&#x3E;Option 1&#x3C;/label&#x3E;
+	&#x3C;/li&#x3E;
+	&#x3C;li class=&#x22;checkbox&#x22;&#x3E;
+		&#x3C;input type=&#x22;checkbox&#x22; id=&#x22;checkbox_2&#x22;&#x3E;
+		&#x3C;label for=&#x22;checkbox_2&#x22;&#x3E;Option 2&#x3C;/label&#x3E;
+	&#x3C;/li&#x3E;
+	&#x3C;li class=&#x22;checkbox&#x22;&#x3E;
+		&#x3C;input type=&#x22;checkbox&#x22; id=&#x22;checkbox_3&#x22;&#x3E;
+		&#x3C;label for=&#x22;checkbox_3&#x22;&#x3E;Option 3&#x3C;/label&#x3E;
+	&#x3C;/li&#x3E;
+	&#x3C;li class=&#x22;checkbox&#x22;&#x3E;
+		&#x3C;input type=&#x22;checkbox&#x22; id=&#x22;checkbox_4&#x22;&#x3E;
+		&#x3C;label for=&#x22;checkbox_4&#x22;&#x3E;Option 4&#x3C;/label&#x3E;
+	&#x3C;/li&#x3E;
+&#x3C;/ul&#x3E;
+
+<span class="com">// Small checkboxes:</span>&#x3C;ul class=&#x22;list-unstyled gc-form&#x22;&#x3E;
+	&#x3C;li class=&#x22;checkbox checkbox-small&#x22;&#x3E;
+		&#x3C;input type=&#x22;checkbox&#x22; id=&#x22;confirmation-checkbox&#x22;&#x3E;
+		&#x3C;label for=&#x22;confirmation-checkbox&#x22;&#x3E;Single option&#x3C;/label&#x3E;
+	&#x3C;/li&#x3E;
+&#x3C;/ul&#x3E;</code></pre>
+		</div>
+	</div>
+	<h3>Radio buttons</h3>
+	<div class="row">
+		<div class="col-md-4">
+			<div class="panel panel-default">
+				<div class="panel-body">
+					<h5 class="mrgn-tp-0">Appearance</h5>
+					<h6>Radio buttons</h6>
+					<ul class="list-unstyled gc-form">
+						<li class="radio">
+							<input id="radio-option-1" type="radio" name="radio-example">
+							<label for="radio-option-1">Option 1</label>
+						</li>
+						<li class="radio">
+							<input id="radio-option-2" type="radio" name="radio-example">
+							<label for="radio-option-2">Option 2</label>
+						</li>
+						<li class="radio">
+							<input id="radio-option-3" type="radio" name="radio-example">
+							<label for="radio-option-3">Option 3</label>
+						</li>
+					</ul>
+				</div>
+			</div>
+		</div>
+		<div class="col-md-4">
+			<h5 class="mrgn-tp-0 text-success"><span class="glyphicon glyphicon-ok-circle"></span> Correct use</h5>
+			<p>Compliance point(s):</p>
+			<ul>
+				<li>Use radio buttons when only one selection can be made.</li>
+			</ul>
+			<h5 class="mrgn-tp-0 text-danger"><span class="glyphicon glyphicon-remove-circle"></span> Incorrect use</h5>
+			<p>Compliance point(s):</p>
+			<ul>
+				<li>Do not use if there is a need for users to select more than one option</li>
+			</ul>
+		</div>
+		<div class="col-md-4">
+			<h5 class="mrgn-tp-0">Code</h5>
+			<pre class="prettyprint prettyprinted" style=""><code><span class="com">// Radio buttons:</span>&#x3C;ul class=&#x22;list-unstyled gc-form&#x22;&#x3E;
```

Comment:
Please use the wet-boew prettify plugin, instead of adding the prettyprint CSS class. 
https://wet-boew.github.io/wet-boew/demos/prettify/prettify-en.html 

Remove the ```style``` empty attribute and I don't think the ```<span class="com">``` is needed. You can put that comment into a paragraph before the preformated text.

Reply:
I tried changing prettyprint to wb-prettify but it didn't appear to work.

The example you gave the link for uses class="prettyprint" as well, so I am not sure how to change this one to use wb-prettify

(<section>
--
  | <h3>HTML</h3>
  | <pre class="prettyprint"><code>&lt;form action="#" method="post"&gt;
  | &lt;div&gt;
  | &lt;label for="data1"&gt;Form input default appearance:&lt;/label&gt;
  | &lt;input name="data1" type="text" id="data1" /&gt;
  | &lt;/div&gt;) 

- code from wb-prettify example documentation

I have removed the style and spans and placed it in a <p>

Reply:
You can simply add the CSS class name ```wb-prettify all-pre``` to any, usually the first, element or on a div once into your page. Then the plugin will apply the syntax colouring to any ````<pre>``` element

```<div class="wb-prettify all-pre"></div>```

Reply:
Thanks! I'll try that out,